doc/api/replication.md

```diff
@@ -0,0 +1,106 @@
+# Thali Cordova Peer Communication API #
+
+## Methods:
+- `StartBroadcasting`
+- `StopBroadcasting`
+- `Connect`
+- `Disconnect`
+- `KillConnection`
+
+Events:
+- `peerAvailabilityChanged`
+- `networkChanged`
+
+***
+
+METHODS:
+
+***
+
+### `StartBroadcasting(deviceName, portNumber, callback)`
+
+This method starts advertising or broadcasting its availability at the following port that PouchDB/Express-PouchDB is listening.  
+
+#### Arguments:
+
+1. `deviceName` : `String` – the device name.
+2. `portNumber` : `Number` – obtained by the user of the code which indicates the port number of our local PouchDB/Express-PouchDB
+3. `callback` : `Function` – must be in the form of the following, `function (err)` where:
+  - `err` : `String` – a string value containing the error if one occurred, else `null`
+
+***
+
+### `StopBroadcasting(callback)`
+
+This method stops advertising or broadcasting of its availability.
+
+#### Arguments:
+
+1. `callback` : `Function` – must be in the form of the following, function (err) where:
+    - `err` : `String` – a string value containing the error if one occurred, else `null`
+
+***
+
+### `Connect(peerIdentifier, callback)`
+
+This method begins a connection to the given peer found during discoverability.  If this is called twice with the same peer identifier, an error will be thrown that it is in process.
```

Comment:
Again, I would use more action oriented language. Something like "This method instructs the native layer to establish a TCP/IP connection to the peer identified by the peerIdentifier, which is obtained via a peerAvailabilityChanged event."

Also shouldn't we specify what the "in process" error is required to look like? Otherwise nobody can tell the difference between a "something went horribly wrong" error and a "hey dummy, you already called this!" error.